lib/epicenter_web/live/place_search_live.html.heex

```diff
@@ -0,0 +1,97 @@
+<div data-page="place-search" data-tid={"#{@case_investigation.tid}"} id="place-page">
+  <section>
+    <header id="subnav">
+      <%= live_redirect to: Routes.profile_path(EpicenterWeb.Endpoint, EpicenterWeb.ProfileLive, @case_investigation.person), data: [role: "back-link"] do %>
+        <%= back_icon() %><span>Back</span>
+      <% end %>
+    </header>
+  </section>
+  <section>
+    <h2>Add place visited</h2>
+    <p>
+      Include workplaces, schools, healthcare facilities, gyms, places of worship, and residence where you spent a meaningful amount of time.
+    </p>
+  </section>
+  <section>
+    <div class="left-right">
+      <form data-role="place-search-form" phx-change="suggest-place">
+        <div id="place-autocomplete">
+          <input
+            aria-autocomplete="list"
+            aria-expanded="true"
+            autocapitalize="none"
+            autocomplete="off"
+            id="place-search-box"
+            name="query"
+            phx-debounce="200"
+            phx-hook="AutocompleteInput"
+            placeholder="name or address"
+            role="combobox"
+            spellcheck="false"
+            type="text"
+            value={"#{@query}"}
```

Comment:
I bet you could clean these up with `fastmod` pretty easily.

Reply:
quite easy to do in vscode as well